### ISSUE TITLE
rtree: issue when reading Char_t typed branches

### ISSUE THREAD

**Issue by rmadar** [go-hep/hep#778]
I am trying to read a `TTree` with a branch of `Char_t` and this crashes with the following error message. I tried with Boolean and string objects and none of them worked. To be honest, I am not sure why this variable is a `Char_t` since it should store just a `bool`.

```
panic: rtree: unknown streamer element: &rdict.StreamerSTL{StreamerElement:rdict.StreamerElement{named:rbase.Named{obj:rbase.Object{ID:0x0, Bits:0x3000000}, name:"This", title:"<Char_t> Used to call the proper TStreamerInfo case"}, etype:500, esize:0, arrlen:0, arrdim:0, maxidx:[5]int32{0, 0, 0, 0, 0}, offset:0, ename:"vector<char>", xmin:0, xmax:0, factor:0}, vtype:1, ctype:1}
```

Let me know if you'd like an test file to try it - I can quickly prepare one.

**Comment by sbinet** [comment]
yes please, having a ROOT file to test things out would help :)

**Comment by rmadar** [comment]
OK, here is a tree with just 2 branches and 10 events. One branch is a `vector<Char_t>` and this other is a `O` (I have no idea what it is, but I thought it could be interesting for you to have it).

https://cernbox.cern.ch/index.php/s/S3y0G5pCXeWEsEO

Let me know if something is not OK.

**Comment by sbinet** [comment]
thanks.
I'll give it a try (but I am kind of off-the-grid between cows and other pastoral sceneries...)

**Comment by rmadar** [comment]
haha, there is no emergency ... no need to spoil your holidays for this :)

**Comment by sbinet** [comment]
```
>>> file[../../testdata/treeCharExample.root]
key[000]: nominal;1 "tree" (TTree)
[000][d_fakeEvent]: false
[000][d_lep_ECIDS]: [1 110]
[001][d_fakeEvent]: false
[001][d_lep_ECIDS]: [1 1]
[002][d_fakeEvent]: false
[002][d_lep_ECIDS]: [110 110]
[003][d_fakeEvent]: false
[003][d_lep_ECIDS]: [1 110]
[004][d_fakeEvent]: false
[004][d_lep_ECIDS]: [1 110]
[005][d_fakeEvent]: false
[005][d_lep_ECIDS]: [110 110]
[006][d_fakeEvent]: false
[006][d_lep_ECIDS]: [1 0]
[007][d_fakeEvent]: false
[007][d_lep_ECIDS]: [1 110]
[008][d_fakeEvent]: false
[008][d_lep_ECIDS]: [1 1]
[009][d_fakeEvent]: false
[009][d_lep_ECIDS]: [1 1]
```

should be fixed w/ #779.

**Comment by rmadar** [comment]
Thanks, it works like a charm in my code too!
